examples/fields.go

```diff
@@ -3,14 +3,14 @@ package main
 
 import (
 	"fmt"
-
+	"github.com/hpe-storage/nimble-golang-sdk/pkg/client"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/client/v1/nimbleos"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/param"
 	"github.com/hpe-storage/nimble-golang-sdk/pkg/service"
 )
 
 func main() {
-	groupService, err := service.NewNsGroupService("1.1.1.1", "xxx", "xxx", "v1", true)
+	groupService, err := service.NewNimbleGroupService(client.WithHost("1.1.1.1"), client.WithUser("xxx"), client.WithPassword("xxx"), client.WithApiVersion("v1"), client.WithoutWaitForAsyncJobs())
```

Comment:
The waitForJobs setting was changed from true to false unintentionally?

Reply:
whoops, this is unintentional. I'm gonna fix it :D thanks!